ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -603,6 +614,35 @@ FAN_SS_VSC_FDPM_DRSM2X:
   - FDPM
   - DRSM2X
 
+FAN_SS_VSC_COM:
+  guid: "4747b38a-c746-4056-bc04-3920d8734fa5"
+  description: "Fan with start/stop status, variable speed control and CO monitoring."
+  is_canonical: true
+  implements:
+  - FAN
+  - SS
+  - VSC
+  - COM
```

Comment:
confirm this isnt control

Reply:
CO sensor is used for alarms, high alarm sends unit into emergency speed control.